src/renderer/ipc/index.tsx

```diff
@@ -51,11 +55,53 @@ function UpdateAvailableHandler(event: IpcRendererEvent, ...[backchannel, update
   );
 }
 
+const listNamespacesForbiddenHandlerDisplayedAt = new Map<string, number>();
+const intervalBetweenNotifications = 1000 * 60; // 60s
+
+function ListNamespacesForbiddenHandler(event: IpcRendererEvent, ...[clusterId]: ListNamespaceForbiddenArgs): void {
+  const lastDisplayedAt = listNamespacesForbiddenHandlerDisplayedAt.get(clusterId);
+  const wasDisplayed = Boolean(lastDisplayedAt);
+  const now = Date.now();
+
+  if (!wasDisplayed || (now - lastDisplayedAt) > intervalBetweenNotifications) {
+    listNamespacesForbiddenHandlerDisplayedAt.set(clusterId, now);
+  } else  {
+    // don't bother the user too often
+    return;
+  }
+
+  const notificationId = `list-namespaces-forbidden:${clusterId}`;
+
+  Notifications.info(
+    (
+      <div className="flex column gaps">
+        <b>Add Accessible Namespaces</b>
+        <p>Cluster <b>{clusterStore.active.name}</b> does not have permissions to list namespaces. Please add the namespaces you have access to.</p>
```

Comment:
How about 


> <p>Cluster <b>{clusterStore.active.name}</b> does not have permissions to list namespaces. Please add the namespaces you have access to in cluster settings.</p>

And button text could then simply be `Add` ? Does it make it any better?

Reply:
I don't know if it makes it better because the button doesn't bring up a dialog but instead it navigates the user to the cluster settings page, scrolls down to the `Accessible Namespaces` section and makes that section easier to find.